### ISSUE TITLE
New, and most powerful turret called Xenifer

### ISSUE THREAD

**Issue by DistributorMadness** [Anuken/Mindustry-Suggestions#3001]
### Describe the content or mechanics you are proposing.

This is a sniper type turret that shoots air and land with 2 holes (obviously for the electrons) with 90 blocks of range it needs 3 impact reactors and 3 thermal pump units of cyrofluid to operate (exactly 3,000 units of electricity per turret [Xenifer]) in a nutshell this turret is silent but packs a punch with 1,010 damage per hit with this turret its DPS is 2, the cost is 800 Thorium 3,000 surge 5,000 copper, 2,000 titanium and 5,000 graphite. The colors are blue, black and white, with the size of 5 x 5 blocks, it has splash around 300 DMG and shock (if it in the center of the blast it will reach 1,010 DMG). it is electric and when it shoots it emits a faint light of blue electrons, and black smoke (blue is the lightning splash and the center of the smoke). the sounds are lancers and arcs combined (weird but better then nothing).
the shape is like foreshadow but blue and black.(!!!Warning: this is not a cheap turret use this if you have enough resources for your own risks, if there is not enough cyrofluid it will explode if the explosion is near the core you will destroy it [You'll probably have a chance to get banned in some servers if you use too much resources]!!!) that is all for the Xenifer turret

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

non.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by acoaco** [comment]
if you draw a cool image anuke may implement it :)

**Comment by begginerr** [comment]
what do you mean my 3 impact reactors and 3 thermal pumps of cryofluid? if you mean what i think you mean, you mean it needs 399 cryofluid (i did the math) to power this thing? also with 2 dps, that would mean that it would shoot every 505 SECONDS!! for the cost is isnt worth it. just use foreshadow if this thing is actually added.

**Comment by BorisA11** [comment]
Two dps is lower than a core ship lol, why would anyone ever use it? And, we dont really need another high power turret, Spectre and Foreshadow are good enough on their own

**Comment by begginerr** [comment]
> Two dps is lower than a core ship lol, why would anyone ever use it? And, we dont really need another high power turret, Spectre and Foreshadow are good enough on their own

wait what? core ships have less than 2 dps?? also yes spectre and foreshadow are really strong anyway

**Comment by Feilong-Mindustry** [comment]
> shoots air and land with 2 holes (obviously for the electrons)
That would be the most schyzophrenic sentence in #suggestions so far.
If you want crazy turrets, check mods. Stop trying to wreck game balance with "cool stuff", please.

**Comment by random-guy1285** [comment]
This suggestion is unintentionally hilarious.

**Comment by SMOLKEYS** [comment]
> non.

_**non.**_ (start a non. reply chain)

**Comment by RealYeet** [comment]
> > non.
> 
> _**non.**_ (start a non. reply chain)

Non

**Comment by oinkymoo** [comment]
this is literally worse than foreshadow, which takes less copper (1000), less surge (300), less total materials (2700), less power (600), less coolant (0), less space (4x4), and deals more damage (1350) and dps (405)

**Comment by begginerr** [comment]
> this is literally worse than foreshadow, which takes less copper (1000), less surge (300), less total materials (2700), less power (600), less coolant (0), less space (4x4), and deals more damage (1350) and dps (405)

i have to agree. blast radius is nice but not enough because of its insanely slow firerate.

**Comment by begginerr** [comment]
if i had to make this thing better, maybe 1000 dps and 3k damage, so 0.3 shots a second so that the cost is actually worth using over foreshadow

**Comment by RealYeet** [comment]
We need this because….?

**Comment by begginerr** [comment]
> We need this because….?

the tower deals 1010 damage, but has 2 dps and uses a lot more than foreshadow, if the tower IS added with 2 dps no one would use it, just look at oinkymoos comment

**Comment by begginerr** [comment]
if you mean why we need this tower, thats a good question. spectre and foreshadow are already good and can handle late game

**Comment by AzhurniyBaklan** [comment]
Close this. Just close this, it's abhorrent.

**Comment by begginerr** [comment]
> Close this. Just close this, it's abhorrent.

i hate to say it..... but i think hes right. i dont even need to explain why this is completely useless.

**Comment by begginerr** [comment]
bro why is this still open, close it already

**Comment by begginerr** [comment]
i hope that you are not dissapointed this has not been added. but really? you want the most powerful turret in the game to have 2 dps? do you even know what dps stands for? it stands for damage per second. if you thought SHOTS per second, then yes. but 2 damage per second? that would mean a HUGE nerf to absolutely everything else.

**Comment by RealYeet** [comment]
It finally died, hallelujah
